src/components/BookItem.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const BookItem = () => (
+  <p>Under Construction</p>
```

Comment:
- [ ] Your BookItem.js should not display the "Under Construction" text, that should be placed in your Categories component, please do so to comply with the project requirements.